GitUI/ScriptsEngine/IScriptHostControl.cs

```diff
@@ -7,5 +7,8 @@ public interface IScriptHostControl
         GitRevision? GetLatestSelectedRevision();
         IReadOnlyList<GitRevision> GetSelectedRevisions();
         Point GetQuickItemSelectorLocation();
+
+        void GoToRef(string? refName, bool showNoRevisionMsg, bool toggleSelection = false);
+        void Refresh();
```

Comment:
XML docs should be added / moved here from RevisionGridControl.

Reply:
`RevisionGridControl.GoToRef` does not have XML docs, so I'm not sure what to copy. The one for `RevisionGridControl.PerformRefreshRevisions` is not really applicable to `IScriptHostControl.Refresh`. I can try writing a specific one for `IScriptHostControl.Refresh`, but I feel like it would look out of place among all the other undocumented methods.